intranet/apps/seniors/tasks.py

```diff
@@ -0,0 +1,12 @@
+from celery import shared_task
```

Comment:
As we manually trigger the deletion of graduating seniors, it does not make sense to make deleting senior destinations a celery task/job.